app/models/user.rb

```diff
@@ -6,7 +6,11 @@ class User < ActiveRecord::Base
   validates :password, :first_name, :last_name, presence: true
   validates :username, length: { in: 2..32, allow_nil: true }
   validates :email, uniqueness: { case_sensitive: false }, presence: true
-  validates :address, presence: true
+  validates :street, presence: true
```

Comment:
I felt more address validations were not necessary because I don't think we can validate properly every single international address.